frontend/src/app/components/enterprise/enterprise-base.component.html

```diff
@@ -0,0 +1,17 @@
+<p *ngIf="!eeTrialService.status">
+  <b>{{ text.become_hero }}</b><br>
+  {{ text.you_contribute }}
+</p>
+
+<div *ngIf="eeTrialService.status === 'mailSubmitted'">
+  <enterprise-trial-waiting>
+    <enterprise-active-trial></enterprise-active-trial>
```

Comment:
I assume that this should show the trial information within the modal? If yes, it doesn't work:
<img width="1097" alt="Bildschirmfoto 2020-03-12 um 13 21 23" src="https://user-images.githubusercontent.com/7457313/76521065-60b1e580-6464-11ea-84be-b46b17ad653e.png">

<img width="1088" alt="Bildschirmfoto 2020-03-12 um 13 19 53" src="https://user-images.githubusercontent.com/7457313/76521105-72938880-6464-11ea-8a13-eb93fdef00aa.png">